nba_api/stats/library/data.py

```diff
@@ -1895,7 +1927,7 @@
     [2738, "Iguodala", "Andre", "Andre Iguodala", True],
     [980, "Ilgauskas", "Zydrunas", "Zydrunas Ilgauskas", False],
     [101148, "Ilic", "Mile", "Mile Ilic", False],
-    [101141, "Ilyasova", "Ersan", "Ersan Ilyasova", True],
+    [101141, "Ilyasova", "Ersan", "Ersan Ilyasova", False],
```

Comment:
```suggestion
    [101141, "Ilyasova", "Ersan", "Ersan Ilyasova", True],
```

Ersan's on the Jazz now.